lib/msal-node/src/client/ClientAssertion.ts

```diff
@@ -69,6 +73,12 @@ export class ClientAssertion {
             [JwtConstants.X5T]: EncodingUtils.base64EncodeUrl(this.thumbprint, "hex")
         };
 
+        if (this.publicCertificate) {
+            Object.assign(header, { 
+                [JwtConstants.X5C]: this.publicCertificate 
+            });
```

Comment:
Since this only modifies one property, can also be:

```suggestion
            header[JwtConstants.X5C] = this.publicCertificate;
```

Reply:
This syntax was actually not working for me, since `publicCertificate` is a `string[]` and not a `string`.

Reply:
Interesting. Not a big deal.